hyperstart.go

```diff
@@ -172,18 +172,23 @@ func (h *hyper) buildNetworkInterfacesAndRoutes(pod Pod) ([]hyperstart.NetworkIf
 		ifaces = append(ifaces, iface)
 
 		for _, r := range endpoint.Properties.Routes {
-			// Skip IPv6 because not supported by hyperstart
-			if r.Dst.IP.To4() == nil {
-				continue
-			}
-
 			gateway := r.GW.String()
 			if gateway == "<nil>" {
 				gateway = ""
 			}
 
+			destination := r.Dst.String()
+			if destination == emptyRouteDest {
+				destination = defaultRouteDest
+			}
+
+			// Skip IPv6 because not supported by hyperstart
+			if destination != defaultRouteDest && r.Dst.IP.To4() == nil {
```

Comment:
@jodh-intel I don't think we have any issues opened for that. Let me add a comment.